src/NuGet.Core/NuGet.ProjectModel/JsonPackageSpecReader.cs

```diff
@@ -380,23 +403,29 @@ private static bool BuildTargetFrameworkNode(PackageSpec packageSpec, KeyValuePa
             return true;
         }
 
-        private static NuGetFramework GetImports(JObject properties)
+        private static List<NuGetFramework> GetImports(JObject properties, PackageSpec packageSpec)
         {
-            NuGetFramework framework = null;
+            List<NuGetFramework> framework = null;
```

Comment:
Methods that return lists usually do not return null. It looks like this will if there is no imports property.